src/js/index.js

```diff
@@ -64,6 +64,7 @@ export const hpe = deepFreeze({
         light: '#BBBBBB',
       },
       'text-xweak': 'text-weak',
+      'text-disabled': 'text-weak',
```

Comment:
What is the long-term purpose to great a new color but use an existing color for it? Is there a plan to adjust `text-weak`?

Reply:
If this is specific to disabled input elements, I think it should be done via `theme.global.input.disabled.color`.

Reply:
There is no plan to adjust text-weak as of what I know, so that is a good point about why we would need to introduce a new color name. Let me follow up with Greg on the motivation here, but maybe we do not need to do this and can just mark `disabled-text` as deprecated and instruct people to use `text-weak`

Reply:
A follow-up here after discussion with Greg -- the intent wasn't that we needed to introduce a new color but instead just clean up color usage where possible. I will update this PR to just add a comment to `disabled-text` that it is deprecated and instead `text-weak` should be used. It will mirror what was done here for `status-disabled`: https://github.com/grommet/grommet-theme-hpe/pull/138